test/dutchExchange-claimSellerFunds.js

```diff
@@ -0,0 +1,124 @@
+
+/* eslint no-console:0, max-len:0, no-plusplus:0, no-mixed-operators:0, no-trailing-spaces:0 */
+/* tulip issuing is tested seperately in dutchExchange-Tulip.js */
+
+const { 
+  eventWatcher,
+  assertRejects,
+} = require('./utils')
+
+const {
+  setupTest,
+  getContracts,
+  getAuctionIndex,
+  waitUntilPriceIsXPercentOfPreviousPrice,
+  setAndCheckAuctionStarted,
+  postBuyOrder,
+  postSellOrder,
+} = require('./testFunctions')
+
+// Test VARS
+let eth
+let gno
+let dx
+
+let contracts
+
+const setupContracts = async () => {
+  contracts = await getContracts();
+  // destructure contracts into upper state
+  ({
+    DutchExchange: dx,
+    EtherToken: eth,
+    TokenGNO: gno,
+  } = contracts)
+}
+const startBal = {
+  startingETH: 90.0.toWei(),
+  startingGNO: 90.0.toWei(),
+  ethUSDPrice: 1008.0.toWei(),
+  sellingAmount: 50.0.toWei(), // Same as web3.toWei(50, 'ether')
+}
+
+contract('DutchExchange - claimSellerFunds', (accounts) => { 
+  const [, seller1, seller2, buyer1, buyer2] = accounts
+  const totalSellAmount2ndAuction = 10e18
+  const totalBuyAmount = 2 * 10e18
+  before(async () => {
+    // get contracts
+    await setupContracts()
+
+    // set up accounts and tokens[contracts]
+    await setupTest(accounts, contracts, startBal)
+
+    // add tokenPair ETH GNO
+    await dx.addTokenPair(
+      eth.address,
+      gno.address,
+      10e18,
```

Comment:
I'm ok with this but should we not try to stick to a standard? e.g `10.0.toWei()` or other